python/lsst/ts/MTAlignment/alignment_model.py

```diff
@@ -85,10 +123,11 @@ async def wait_for_ready(self):
             # even if the tracker reports ready.
             # If that is false, move this line to the end of the while block
             await asyncio.sleep(0.5)
-            if reply == "READY":
+            if reply.startswith("READY"):
```

Comment:
Would it be worth logging the `reply`? Also, maybe change `await asyncio.sleep(0.5)` in the line above with some variable instead of a hard coded value?

Also, looks like there's no timeout here, so we could get in an infinite loop.. hehe..

Reply:
All replies are already logged. Timeout is likely a good idea, but I'll have to inquire as to what might be a reasonable value. I have no idea where the 0.5 seconds comes from and am a bit reluctant to mess with it, but I'll also inquire about that.